index.html

```diff
@@ -179,37 +216,37 @@ <h4>Skills</h4>
         </div> 
     </div>
     </section>
-    <Section class="contact">
+    <Section class="contact-form">
+        <div class="contact">
         <h3>
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h3>
         <div class="form-section">
             <form action="https://formspree.io/f/myyvjjng" method="post" id="form">
                 <div class="form-input">
-                    <input type="text"  id="name" name="user_name" placeholder="Full Name" maxlength="30" required>
-                    <input type="email"  id="email" name="user_email" placeholder="Email Address" required>
+                    <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    <input type="email"  id="email" name="user_email"  placeholder="Email address" required>
                     <textarea type="text"  id="message" name="user_message"  maxlength="500" placeholder="Enter text here" required></textarea>
                 </div>
-                <div class="back-img4"></div>
                 <div class="get-in-touch-btn">
                     <button class="form-btn" type="submit">Get in touch </button>
-                </div>
-            
+                </div> 
             </form>
         </div>
+    </div>
+    <div class="back-img4"></div>
```

Comment:
- [ OPTIONAL ] I suggest removing the space after the form so kindly remove it because it too big and different than Figma design so it is better to stick with the Figma

- Your design
![image](https://user-images.githubusercontent.com/58553711/199730179-72d0dd76-2273-447f-b1bf-0ffda7cfb097.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/199730308-c12b30c4-cc7b-4f96-a450-a13824df912f.png)